packages/web-components/src/components/va-textarea/va-textarea.scss

```diff
@@ -8,6 +8,7 @@
 @use 'usa-textarea/src/styles/usa-textarea';
 @use 'usa-character-count/src/styles/_usa-character-count';
 
+@import '../../mixins/accessibility.css';
```

Comment:
I think we might be able to remove this import too. It imports `.sr-only` but we should probably be replacing that with the USWDS `.usa-sr-only` class now using this import:

```
@use 'uswds-helpers/src/styles/usa-sr-only';
```